
jump_distance: 1.5 # meters
speed_up_distance: 4.5 # meters; needs to be +2m larger than actual distance
                        # use 3.5m for <= 1.0m jump; use 4.5m for 1.5m jump
ramp_base: 24.0
ramp_height: 12.0
ramp_units: 'inch'

stop_speed: 6000
slow_speed: 6100
slow_time: 0.5     # seconds

#### OLD YAML FILE STUFF

# jump_speed: 6630
# stop_speed: 6000
# forward_time: 1.4

# measured dist | calculated dist | forward time | speed
# -------------------------------------------------------
#    1.0m             1.3m              1.4s       6630
#    0.5m             0.8m              1.7s       6465
# --------------------------------------------------------
